*** tests/A-02-13-M-ok.out	2018-11-18 16:01:43.000000000 +0000
--- tests/A-02-13-M-ok.outhyp	2018-12-04 12:00:29.518673509 +0000
***************
*** 11,20 ****
  4 - Procurar pessoa
  0 - Sair
  Escolha uma opção: DOCENTE|100100|987654321|Ana Maria
- * Informática - Análise e Síntese de Algoritmos
- * Informática - Engenharia de Software
  * Matemática - Algebra Linear
  * Matemática - Compiladores
  Portal Pessoal
  1 - Mostrar pessoa
  2 - Alterar número de telefone
--- 11,20 ----
  4 - Procurar pessoa
  0 - Sair
  Escolha uma opção: DOCENTE|100100|987654321|Ana Maria
  * Matemática - Algebra Linear
+ * Informática - Análise e Síntese de Algoritmos
  * Matemática - Compiladores
+ * Informática - Engenharia de Software
  Portal Pessoal
  1 - Mostrar pessoa
  2 - Alterar número de telefone
