*** tests/A-02-05-M-ok.out	2018-11-05 18:01:01.000000000 +0000
--- tests/A-02-05-M-ok.outhyp	2018-12-04 12:00:24.114478151 +0000
***************
*** 11,22 ****
  4 - Procurar pessoa
  0 - Sair
  Escolha uma opção: DOCENTE|100100|987654321|Ana Maria
- * Electrónica - Sistemas Operativos
  * Informática - Análise e Síntese de Algoritmos
! * Informática - Compiladores
  * Informática - Engenharia de Software
  * Informática - Sistemas Operativos
- * Matemática - Compiladores
  Portal Pessoal
  1 - Mostrar pessoa
  2 - Alterar número de telefone
--- 11,20 ----
  4 - Procurar pessoa
  0 - Sair
  Escolha uma opção: DOCENTE|100100|987654321|Ana Maria
  * Informática - Análise e Síntese de Algoritmos
! * Matemática - Compiladores
  * Informática - Engenharia de Software
  * Informática - Sistemas Operativos
  Portal Pessoal
  1 - Mostrar pessoa
  2 - Alterar número de telefone
