*** tests/A-09-02-M-ok.out	2018-11-04 11:46:48.000000000 +0000
--- tests/A-09-02-M-ok.outhyp	2018-12-04 12:00:48.923259840 +0000
***************
*** 11,17 ****
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
--- 11,18 ----
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Ver alunos de disciplina leccionada: Operação inválida: A disciplina 'The Dark Side' não existe ou não é válida para o utilizador em causa.
! Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
