*** tests/A-10-34-M-ok.out	2018-12-05 18:33:50.292541692 +0000
--- tests/A-10-34-M-ok.outhyp	2018-12-06 10:30:59.435218501 +0000
***************
*** 43,50 ****
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Criar inquérito: Operação inválida: O projecto 'P2' da disciplina 'D1' já tem um inquérito associado.
! Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
--- 43,49 ----
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
