*** tests/A-10-29-M-ok.out	2018-12-05 18:33:50.016540387 +0000
--- tests/A-10-29-M-ok.outhyp	2018-12-06 10:30:57.631203685 +0000
***************
*** 32,45 ****
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Ver resultados de inquérito: Operação inválida: O projecto 'P2' da disciplina 'D1' não tem inquérito associado.
! Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Ver resultados de inquérito: Operação inválida: O projecto 'P3' não existe na disciplina 'D1'.
! Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
--- 32,43 ----
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
