*** tests/A-10-14-M-ok.out	2018-12-02 16:46:40.000000000 +0000
--- tests/A-10-14-M-ok.outhyp	2018-12-05 17:54:01.525039587 +0000
***************
*** 17,24 ****
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1 (por abrir)
! Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
--- 17,23 ----
  4 - Ver alunos de disciplina leccionada
  5 - Ver resultados de um inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Docente
  1 - Criar projecto
  2 - Fechar projecto
  3 - Ver submissões de um projecto
