*** tests/A-10-40-M-ok.out	2018-12-02 23:27:43.000000000 +0000
--- tests/A-10-40-M-ok.outhyp	2018-12-06 22:44:42.614912986 +0000
***************
*** 20,26 ****
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
--- 20,27 ----
  5 - Finalizar inquérito
  6 - Mostrar inquéritos de uma disciplina
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: Fechar inquérito: Operação inválida: Problema ao fechar o inquérito do projecto 'P1' da disciplina 'D1'.
! Portal do Delegado
  1 - Criar inquérito
  2 - Cancelar inquérito
  3 - Abrir inquérito
***************
*** 40,46 ****
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1 (fechado)
  Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
--- 41,47 ----
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1
  Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
***************
*** 81,87 ****
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1 (aberto)
  Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
--- 82,88 ----
  2 - Preencher inquérito
  3 - Ver resultados de inquérito
  0 - Sair
! Escolha uma opção: Nome da disciplina: Nome do projecto: D1 - P1
  Portal do Estudante
  1 - Entregar projecto
  2 - Preencher inquérito
