*** tests/A-10-30-M-ok.out	2018-12-02 16:46:44.000000000 +0000
--- tests/A-10-30-M-ok.outhyp	2018-12-06 22:44:41.126916217 +0000
***************
*** 5,11 ****
  5 - Portal do Estudante
  6 - Portal do Delegado
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Pode preencher inquérito do projecto P1 da disciplina D1
  Menu principal
  1 - Abrir
  2 - Guardar
--- 5,11 ----
  5 - Portal do Estudante
  6 - Portal do Delegado
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Pode preencher inquérito do projeto P1 da disciplina D1
  Menu principal
  1 - Abrir
  2 - Guardar
